jekyll/_cci2/test-splitting-tutorial.adoc

```diff
@@ -113,19 +156,62 @@ steps:
     name: Run tests
     command: yarn test
 ----
+
+. After you've made these changes, go ahead and click the btn:[Commit and Run]. This will commit the changes in a new feature branch called `circleci-project-setup` and trigger a new pipeline. 
++
+image::test-splitting-first-pipeline.png[Successful pipeline run]
++
+Feel free to take a look at the steps run in the pipeline by expanding the green Success status and opening the `build-and-test` job.
++
+image::test-splitting-first-setup-steps.png[Steps run successfully within the job]
+--
+
+[.tab.add_project.GitLab]
+--
+. Import the project code into GitLab, using the repo URL: `https://github.com/CircleCI-Public/circleci-react-test-splitting-tutorial`
+
+. Open the link:https://app.circleci.com[CircleCI web app]. Make sure you are in the correct organization (`\https://app.circleci.com/pipelines/circleci/<project-slug>`), then navigate to **Projects**.
```

Comment:
I don't think the URL will help people here, not sure a standalone user can find <project-slug> prior to accessing the app anyway